storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -191,6 +191,12 @@ static regex_list_t regex_exclude_list;
 static hash_table_t *tables_include_hash = NULL;
 static hash_table_t *tables_exclude_hash = NULL;
 
+/* map of schema name and schema id */
+static std::map<std::string, int> dd_schema_map;
+
+/* map of <schema id, name> and key */
+static std::map<std::pair<int, std::string>, int> dd_table_map;
```

Comment:
sometimes the ids are declared as uint64 and sometimes as int.

Better to use uint64 everywhere

Reply:
fixed